docs/kf.dev/content/en/docs/developers/reference-apps/link-shortener/_index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Link Shortener App "
+linkTitle: "Link Shortener App"
+weight: 20
+description: >
+  Learn how to deploy the Link Shortener reference application originally built
+  for Cloud Foundry and the Google Cloud service broker.
+---
+
+This example Spring application is an enterprise-ready link-shortener.
+The URL shortener has the following behaviour:
```

Comment:
s/behaviour/behavior

this is 'merica!! 🇺🇸

Reply:
Done